exporter/splunkhecexporter/README.md

```diff
@@ -28,6 +28,13 @@ The following configuration options can also be configured:
 - `max_content_length_logs` (default: 2097152): Maximum log data size in bytes per HTTP post limited to 2097152 bytes (2 MiB).
 - `splunk_app_name` (default: "OpenTelemetry Collector Contrib") App name is used to track telemetry information for Splunk App's using HEC by App name.
 - `splunk_app_version` (default: Current OpenTelemetry Collector Contrib Build Version): App version is used to track telemetry information for Splunk App's using HEC by App version.
+- `hec_to_otel_attrs/source` (default = 'com.splunk.source'): Specifies the mapping of a specific unified model attribute value to the standard source field of a HEC event.
```

Comment:
I see that the naming was discussed and changed in the past, but I still find the current naming confusing. In Splunk docs these are called "metadata" so perhaps  we call this setting `hec_metadata_from_otel_attrs`?

Reply:
done